com.todesk.ToDesk.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: com.todesk.ToDesk
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: todesk.sh
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --own-name=com.todesk.toc.ui
+  - --device=dri
+  # Share files between local and remote.
+  - --filesystem=xdg-download
+separate-locales: false
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: libkmod
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://mirrors.edge.kernel.org/pub/linux/utils/kernel/kmod/kmod-32.tar.xz
+        sha256: 630ed0d92275a88cb9a7bf68f5700e911fdadaf02e051cf2e4680ff8480bd492
+  - name: todesk 
+    buildsystem: simple
+    build-commands:
+      - install -D -m 755 -t /app/bin/ apply_extra todesk.sh
+      - install -D -t /app/lib/ /usr/lib/x86_64-linux-gnu/{libbfd-*.so,libsframe.so.1}
+      - install -D -t /app/bin/ /usr/bin/ar
+      - mkdir -p /app/share/icons/ && cp -a hicolor /app/share/icons/
+      - install -D -t /app/share/applications/ com.todesk.ToDesk.desktop
+      - install -D -t /app/share/metainfo/ com.todesk.ToDesk.metainfo.xml
+    only-arches:
+      - "x86_64"
+    sources:
+      - type: script
+        commands:
+          - ar x todesk-amd64.deb
+          - tar -xf data.tar.xz
+          - mv opt/todesk .
+          - rm -rf opt usr etc *.tar.* debian-binary *.deb
+          - uuidgen > uuid
+        dest-filename: apply_extra
+      - type: extra-data
+        filename: todesk-amd64.deb
+        url: https://dl.todesk.com/linux/todesk-v4.7.2.0-amd64.deb
+        sha256: bfb5695d715568a23df51a735167c073a784ecd1c625e16b35e5136d5b97fb28
+        size: 63204994
+        x-checker-data:
+          type: html
+          url: https://www.todesk.com/linux.html
+          pattern: <a type="button" href="(https://dl.todesk.com/linux/todesk-v([\d\.]+\d)-amd64.deb)" class="button" .*>立即下载</a>
+      - type: file
+        path: todesk.sh
+      - type: dir
+        path: icons
```

Comment:
Install just the 512px one delete rest of the icons.

Reply:
Done.

Reply:
don't use dir source. There is just one file just list it as such.

Reply:
@hfiguiere Done.